modules/serverless-rn-1-14-0.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-14-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.14.0
+
+[id="new-features-1-14-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.20.0.
+* {ServerlessProductName} uses Knative Eventing 0.20.0.
+* {ServerlessProductName} now uses Kourier 0.20.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.20.0.
+* {ServerlessProductName} now uses Knative Kafka 0.20.0.
+* Apache Kafka on {ServerlessProductName} is now Generally Available (GA).
++
+[IMPORTANT]
+====
+The `v1beta1` version of the API for Apache Kafka on {ServerlessProductName} is now now supported. Do not use the `v1alpha1` version of the API, as this is now deprecated.
```

Comment:
`now now supported` (twice there is a `now`)

More of a comment on the message again. Since we have first added the KafkaChannel to the operator, it had always the beta1 API. I'd not use the word "now", b/c that sounds to me like it was never there before.

It's more that we want to give users a heads up to really stick to the v1beta1 API, and not bother with the deprecated v1alpha1 APIs

Reply:
I'm not sure I'd add this to the release notes in that case if it's not something new; unless there's something a user has to do to upgrade to the new version, let's assume that they are using the latest version and make sure this is reflected in any YAML examples for Kafka? For example, we don't have notes on any other APIs re only using the latest version, we just assume AFAIK that folks use v1 or whatever we currently support / provide.